cmd/curio/guidedsetup/guidedsetup.go

```diff
@@ -164,6 +181,23 @@ func SetupLanguage() (func(key message.Reference, a ...interface{}) string, func
 		}
 }
 
+func newOrMigrate(d *MigrationData) {
+	i, _, err := (&promptui.Select{
+		Label: d.T("Choose if you with to create a new miner or migrate from existing Lotus-Miner"),
+		Items: []string{
+			d.T("Migrate from existing Lotus-Miner"),
+			d.T("Create a new miner")},
```

Comment:
```suggestion
			d.T("Create a new miner with info from `%s` ", "sptool create miner")},
```
If that's where it will be.

Reply:
We have a separate command for creating config. We use that in case guided-setup fails. We don't need `sptool create-miner` for the guided-setup.